packages/charts/react-charts/stories/src/AreaChart/AreaChartDefault.stories.tsx

```diff
@@ -1,6 +1,16 @@
 import * as React from 'react';
 import { AreaChart } from '@fluentui/react-charts';
-import { Switch, Field, Radio, RadioGroup } from '@fluentui/react-components';
+import { Switch, Field, Radio, RadioGroup, makeStyles, tokens, mergeClasses } from '@fluentui/react-components';
+import { HighContrastSelector } from '../../../library/src/utilities/utilities';
+
+const useStyles = makeStyles({
+  svgTooltip: {
+    fill: tokens.colorNeutralBackground2,
```

Comment:
> fill: tokens.colorNeutralBackground2, [](http://example.com/codeflow?start=3&length=38)

if you merge the props and internal defined class, you will need to override only the fill property